### PR TITLE
[#2PlaysAMonth]: Basic calculator is made with react.js

### PR DESCRIPTION
**Pull request by bahnisikhadhar**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes#1006

## Type of change

Please delete options that are not relevant.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ yes] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/plays/basic-calculator/BasicCalculator.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  // Your Code Start below.
+  const [{ currentOperand, previousOperand, operation }, dispatch] = useReducer(reducer, {})
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
+        {/* Your Code Starts Here */}
+        <div className="outer_container">
```
Comment (atapas) [inline src/plays/basic-calculator/BasicCalculator.js:140]: The line # 140 to 174 can go to a separate component.

File: src/plays/basic-calculator/Readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Language: js
+- Level: Beginner
+
+## Creator Information
+
+- User: bahnisikhadhar
+- Gihub Link: https://github.com/bahnisikhadhar
+- Blog: 
+- Video:
```
Comment (atapas) [inline src/plays/basic-calculator/Readme.md:15]: Kindly record a video about your demo on stack stream.

File: src/plays/basic-calculator/styles.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/* enter stlyes here */
+*{
```
Comment (atapas) [inline src/plays/basic-calculator/styles.css:2]: No * override allowed.

File: src/plays/basic-calculator/styles.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  .outer_container{
```
Comment (atapas) [inline src/plays/basic-calculator/styles.css:7]: Use a unique class name to prefix all the styles in this file. You already got a class name `calculator_grid`, you may want to use that

**Review by atapas — CHANGES_REQUESTED** [review]
@bahnisikhadhar Review comments. Please take care.

File: src/plays/basic-calculator/BasicCalculator.js (hunk 1 above)
Comment (bahnisikhadhar) [reply src/plays/basic-calculator/BasicCalculator.js:140]: ok I am looking into this

File: src/plays/basic-calculator/Readme.md (hunk 2 above)
Comment (bahnisikhadhar) [reply src/plays/basic-calculator/Readme.md:15]: ok will do that

File: src/plays/basic-calculator/styles.css (hunk 3 above)
Comment (bahnisikhadhar) [reply src/plays/basic-calculator/styles.css:2]: okay, will fix it

File: src/plays/basic-calculator/styles.css (hunk 4 above)
Comment (bahnisikhadhar) [reply src/plays/basic-calculator/styles.css:7]: ok sure, Thank you

**Comment by atapas** [comment]
@bahnisikhadhar 

Also, your play doesn't have a cover image.. please check create-play readme file to how to add that.

**Comment by atapas** [comment]
> @bahnisikhadhar
> 
> Also, your play doesn't have a cover image.. please check create-play readme file to how to add that.

@bahnisikhadhar Have you taken care of these as well?

**Comment by bahnisikhadhar** [comment]
Yes, I have updated my cover image

**Comment by bahnisikhadhar** [comment]
> > @bahnisikhadhar
> > Also, your play doesn't have a cover image.. please check create-play readme file to how to add that.
> 
> @bahnisikhadhar Have you taken care of these as well?

Yes, I have added cover image

**Review by atapas — APPROVED** [review]

**Comment by atapas** [comment]
@bahnisikhadhar It is ready and merging it. It will be available on the reactplay site after the build(about 10 mins from now)

Go ahead and share about your contributions. Once you do so, feel free to Tag ReactPlay and the #2PlaysAMonth.

**Comment by bahnisikhadhar** [comment]
> @bahnisikhadhar It is ready and merging it. It will be available on the reactplay site after the build(about 10 mins from now)
> 
> Go ahead and share about your contributions. Once you do so, feel free to Tag ReactPlay and the #2PlaysAMonth.

yes sure, Thank you so much @atapas

**Pull request merged**